src/content/contributing/code-review/_index.en.md

```diff
@@ -6,44 +6,39 @@ weight: 30
 
 ## Code Review Guide
 
-This guide is meant to facilitate Submariner code review by sharing norms, best
-practices, and useful patterns.
+This guide is meant to facilitate Submariner code review by sharing norms, best practices, and useful patterns.
 
-Submariner follows the [Kubernetes Code Review Guide][kube code review guide]
-wherever relevant. This guide collects the most important highlights of the
-K8s process and adds Submariner-specific extensions.
+Submariner follows the [Kubernetes Code Review Guide][kube code review guide] wherever relevant. This guide collects the most important
+highlights of the K8s process and adds Submariner-specific extensions.
 
-### Two non-author approvals required
+### Two non-author Committer approvals required
 
-Pull Requests to Submariner require two non-author code review approvals.
-
-At least one approval must be from a Committer to the relevant part of the code
-base, as defined by the CODEOWNERS file at the root of the repository.
+Pull Requests to Submariner require two approvals from a Committer to the relevant part of the code base, as defined by the CODEOWNERS file
+at the root of each repository and the [Community
+Membership/Committers](http://localhost:1313/contributing/community-membership/#committers) process.
 
 ### No merge commits
 
 Kubernetes recommends [avoiding merge commits][merge commits].
 
-Use `git fetch` and `git rebase` to avoid them.
+Our current GitHub configuration should automatically remove merge commits.
```

Comment:
```suggestion
Our current GitHub configuration should automatically remove merge commits.
```

With our current GitHub setup, pull requests are liable to include merge commits temporarily: whenever a PR is updated through the UI, GitHub merges the target branch into the PR. However, since we merge PRs by either squashing or rebasing them, those merge commits disappear from the series of commits which ultimately ends up in the target branch.